src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -95,7 +95,19 @@ module sonic-portchannel {
                     mandatory true;
                     type stypes:admin_status;
                 }
-            } /* end of list PORTCHANNEL_LIST */
+
+                leaf lacp_key {
+                    /* lacp key should be either auto or a integer in the range of 1 to 65535 */
+                    type union {
+                        type string {
+                            pattern "auto";
```

Comment:
can you add description for what does auto mean here? it is not clear what it means.